hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/config/GooglePubSubSenderProperties.java

```diff
@@ -19,6 +19,8 @@ public class GooglePubSubSenderProperties {
 
     private String transportChannelProviderAddress = "integration";
 
+    private boolean includeMoreAttributes = false;
```

Comment:
This name seems to be too general - without additional documentation or looking into code I can't tell what it does. Unfortunately, I can't find any short and neat name for that. Do we need to have this property? Maybe we can assume that for now a minimal set of headers is always enough. WDYT?

Reply:
Agree. I'll change it in few minutes ;)